src/functionalTest/java/uk/gov/hmcts/reform/draftstore/features/store/Draft.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.draftstore.features.store;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class Draft {
+
+    private final JSONObject json;
+
+    public Draft(JSONObject json) {
+        this.json = json;
+    }
+
+    public static Draft empty() throws JSONException {
+        return create("", new JSONObject());
+    }
+
+    public static Draft create(String type, Object document) throws JSONException {
```

Comment:
this method is always called with the same parameters.
Do we need this object?
Can we simply supply json string instead?